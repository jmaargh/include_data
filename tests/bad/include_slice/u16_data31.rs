fn main() {}

// can't make a u16 array of 31 bytes
static BYTES_31: &[u16] = include_data::include_slice!(u16, "../../test_data/binary_31");
