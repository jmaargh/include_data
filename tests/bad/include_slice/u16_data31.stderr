error[E0080]: could not evaluate static initializer
 --> tests/bad/include_slice/u16_data31.rs:4:27
  |
4 | static BYTES_31: &[u16] = include_data::include_slice!(u16, "../../test_data/binary_31");
  |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the evaluated program panicked at 'Included file size is not divisible by target type size', $DIR/tests/bad/include_slice/u16_data31.rs:4:27
  |
  = note: this error originates in the macro `$crate::panic::panic_2021` which comes from the expansion of the macro `include_data::include_slice` (in Nightly builds, run with -Z macro-backtrace for more info)
